test/test.js

```diff
@@ -33,44 +33,81 @@ describe("TeamCity build number", function() {
 	});
 
 	describe("Name Matches Convention", function() {
+		var branchNamingConventionRegex = /^[A-Z]{2,10}-\d+-[A-Z]+[A-Za-z0-9-_]+$/, 
```

Comment:
I think these might be better as named exports from the main module so we can re-use them